PC/msvcrtmodule.c

```diff
@@ -253,6 +253,8 @@ msvcrt_getch_impl(PyObject *module)
     return ch;
 }
 
+#ifndef MS_WINDOWS_GAMES
```

Comment:
```suggestion
#ifndef MS_WINDOWS_NON_DESKTOP
```
The CRT doesn't implement the wide-character console I/O functions for non-desktop builds.